pro/src/ui-kit/Button/ButtonLink.tsx

```diff
@@ -24,11 +26,17 @@ const ButtonLink = ({
   const classNames = cn(
     styles['button'],
     styles[`button-${variant}`],
+    isDisabled ? styles[`button-disabled`] : null,
```

Comment:
```suggestion
    { [styles[`button-disabled`]] : isDisabled },
```

`classnames` permet d'avoir des classes de manière conditionnel sour la forme `{ 'maClassCss': true }` ou `{ 'maClassCss': false }`